node/service/src/service/pangolin.rs

```diff
@@ -67,15 +67,18 @@ use dp_rpc::{FilterPool, PendingTransactions};
 use drml_common_primitives::{AccountId, Balance, Hash, Nonce, OpaqueBlock as Block, Power};
 use drml_rpc::{
 	pangolin::{FullDeps, LightDeps},
-	BabeDeps, DenyUnsafe, GrandpaDeps, SubscriptionTaskExecutor,
+	BabeDeps, GrandpaDeps, RpcConfig, SubscriptionTaskExecutor,
 };
 use pangolin_runtime::RuntimeApi;
 
 sc_executor::native_executor_instance!(
 	pub Executor,
 	pangolin_runtime::api::dispatch,
 	pangolin_runtime::native_version,
-	frame_benchmarking::benchmarking::HostFunctions,
+	(
+		frame_benchmarking::benchmarking::HostFunctions,
+		dp_evm_trace_ext::dvm_ext::HostFunctions
+	),
```

Comment:
@HackFisher We introduce this `dvm_ext::HostFunctions`. I am not totally sure if a forced upgrade is needed. We need more tests in the local env or pangolin network.

Reply:
For on-chain wasms we do not have the trace feature enabled right, so it might not use the new introduce host functions right, possibly will not cause older nodes broken?   (Not sure there are host function/runtime-interface definitions embedded in runtime, then there could be checks between node and wasm)

Trace node will definitely require latest version?

We need to try and test to help us understand these questions? I can create a question issue later.